hugegraph-llm/src/hugegraph_llm/operators/kg_construction_task.py

```diff
@@ -60,16 +59,17 @@ def fetch_graph_data(self):
         return self
 
     def chunk_split(
-            self,
-            text: Union[str, List[str]],  # text to be split
-            split_type: Literal["paragraph", "sentence"] = "paragraph",
-            language: Literal["zh", "en"] = "zh"
+        self,
+        text: Union[str, List[str]],  # text to be split
+        split_type: Literal["document", "paragraph", "sentence"] = "document",
```

Comment:
Why is the default value changed to `document`, does it have a better effect?

Reply:
> Why is the default value changed to `document`, does it have a better effect?

handle it in next PR (we need refactor the split logic for better effect)